sat/cli/status/status_module.py

```diff
@@ -553,12 +552,22 @@ def rows(self):
                     try:
                         component_session = bos_client.get_session(session_id)
                     except APIError as err:
-                        LOGGER.warning('Could not retrieve BOS session for component %s: %s',
-                                       component_xname, err)
+                        LOGGER.debug('Could not retrieve BOS session for component %s: %s',
+                                     component_xname, err)
                     self._cached_bos_sessions[session_id] = component_session
                 if component_session is None:
                     continue
 
+                # Track missing sessions
+                if session_id in self.missing_sessions:
+                    self.missing_sessions[session_id] += 1
+                else:
+                    self.missing_sessions[session_id] = 1
+
+                for session_id, count in self.missing_sessions.items():
+                    LOGGER.debug('Session id %s which applies to %d nodes is missing',
+                                 session_id, count)
```

Comment:
Due to the placement of this loop inside the loop that is looping over each BOS component, this message gets printed out many many times as it's being built up when it should instead be printed out just once for each missing session after iterating over every BOS component.

Here's what I see when I ran this code, for example:

```
ncn-m001:~/haasken # sat --loglevel debug status --bos-fields
...
DEBUG: Session id 99194948-b639-489b-8d9b-a41f4fce4187 which applies to 1 nodes is missing
...
DEBUG: Session id 99194948-b639-489b-8d9b-a41f4fce4187 which applies to 2 nodes is missing
...
DEBUG: Session id 99194948-b639-489b-8d9b-a41f4fce4187 which applies to 16 nodes is missing
DEBUG: Session id 17f89b96-6193-4bc1-9888-a0f99a106ea7 which applies to 1 nodes is missing
DEBUG: Session id dad4be51-88c0-4850-905d-77d9b4c1d368 which applies to 1 nodes is missing
DEBUG: Session id bf55ca42-6a5f-402a-be56-17cd54031533 which applies to 1 nodes is missing
...
DEBUG: Session id 99194948-b639-489b-8d9b-a41f4fce4187 which applies to 17 nodes is missing
DEBUG: Session id 17f89b96-6193-4bc1-9888-a0f99a106ea7 which applies to 1 nodes is missing
DEBUG: Session id dad4be51-88c0-4850-905d-77d9b4c1d368 which applies to 1 nodes is missing
DEBUG: Session id bf55ca42-6a5f-402a-be56-17cd54031533 which applies to 1 nodes is missing
...
DEBUG: Session id 99194948-b639-489b-8d9b-a41f4fce4187 which applies to 20 nodes is missing
DEBUG: Session id 17f89b96-6193-4bc1-9888-a0f99a106ea7 which applies to 1 nodes is missing
DEBUG: Session id dad4be51-88c0-4850-905d-77d9b4c1d368 which applies to 1 nodes is missing
DEBUG: Session id bf55ca42-6a5f-402a-be56-17cd54031533 which applies to 1 nodes is missing
```

I omitted many messages for brevity, but you can see the problem.